src/auslib/web/admin/swagger/api_v2.yml

```diff
@@ -463,8 +478,9 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ReleaseModifyResponse'
```

Comment:
Not sure about this one, but having a non-unique key in an object is not valid YAML